cli/src/check-license.ts

```diff
@@ -57,6 +42,28 @@ export async function checkLicense(packagePath?: string): Promise<void> {
     } while (answer === 'help');
 }
 
+export async function isLicenceOk(packagePath?: string): Promise<boolean> {
+    if (await hasLicenseFile(packagePath)) {
+        // The extension has a LICENSE file that can be packaged
+        return true;
+    }
+    const manifest = await readManifest(packagePath);
```

Comment:
Now the manifest is read twice during publishing. It would be great to avoid that.

Reply:
@spoenemann I've just updated the function to avoid this behaviour.